proto/v1beta1/spdx.proto

```diff
@@ -256,12 +258,138 @@ message PackageInfoOccurrence {
   // A place for the SPDX file creator to record any general
   // comments about the package being described
   string comment = 6;
+  // The type of package: OS, MAVEN, GO, GO_STDLIB, etc.
```

Comment:
do we need the same field in the Note and Occurrence?

Reply:
Having the same field in the Occurrence allows clients to get most of the info by calling ListOccurrences, and not needing to call ListNotes. This has a few benefits, such as simpler client code, better UX performance, as well as less load on the API.

Reply:
I added comment "Output only" for each field that is copied from a Note.

Reply:
great, thanks!